src/components/sw360/Table/Table.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Helio Chissini de Castro, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+import * as React from 'react'
+import { Component, createRef, RefObject } from 'react'
+import { Config, Grid } from 'gridjs'
+import { Form } from 'react-bootstrap'
+
+const defaultOptions = {
+    pagination: { limit: 10 },
+    search: false,
+    selector: true,
+    sort: true,
+}
+
+interface TableProps extends Partial<Config> {
+    selector?: boolean
+}
+
+class Table extends Component<TableProps, unknown> {
```

Comment:
Use Function Component instead of Class Component

Reply:
In this specific case we do need to use class instead of function due to nature of original gridjs ( non react ) component.